lib/roger/partition/consumer.ex

```diff
@@ -38,6 +38,16 @@ defmodule Roger.Partition.Consumer do
     end
   end
 
+  @doc """
+  Pauses all running queues on current consumer
+  """
+  @spec pause_all(String.t) :: :ok | nil
+  def pause_all(partition_id) do
+    if is_alive?(partition_id) do
+      GenServer.call(GProc.via(name(partition_id)), {:pause_all})
```

Comment:
Shouldn't this be an async `cast`? Since the return value doesn't really matter I believe.

Reply:
Yeah I wasn't sure about this one myself advantage on this is you know the queues are truly paused when you continue on in your code.